tailwind.config.js

```diff
@@ -0,0 +1,68 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {
+      animation: {
+        dropdown: "dropdown 0.3s ease 0s 1 normal forwards",
+      },
+      boxShadow: {
+        pictures: "-0.625rem -0.0313rem 5.625rem 0.4375rem rgba(0, 0, 0, 0.12)",
+      },
+      keyframes: {
+        dropdown: {
+          "0%": { transform: "translateY(-100px)" },
+          "100%": {
+            position: "relative",
+            transform: "translateY(0px)",
+          },
+        },
+      },
+      screens: {
+        desktop: "90rem",
+        tablet: "64rem",
+        ipad: "52.125rem",
+        mobile: "24.375rem",
+      },
+
+      colors: {
+        purple: "#5602E0",
+        secondary: {
+          50: "#E4E4E4",
+          100: "#9A9A9A",
+        },
+      },
+      fontFamily: {
+        sans: ["Neufile Grotes", "arial"],
+      },
+      maxWidth: {
+        90: "90rem",
+      },
+      margin: {
+        18: "4.4375rem",
+      },
+      padding: {
+        4.5: "1.1219rem",
+        5.5: "1.3256rem",
+      },
+      letterSpacing: {
+        thin: "-0.015rem",
```

Comment:
you don't need negative ` thin: "-0.015rem",` you can define this  without `-` and in your design add `-` before the utility class